### PR TITLE
Add hvnSimFit

### PR DESCRIPTION
**Pull request by Libeiyu**

### CODE REVIEW

**Comment by Libeiyu** [comment]
Part of the code used for local analysis in run2 requires a hist like this.

File: run3/flow/get_vn_vs_mass.py [hunk 1]
```diff
@@ -353,6 +359,10 @@ def get_vn_vs_mass(fitConfigFileName, centClass, inFileName,
             vnFitter[iPt].SetInitialGaussianSigma(fitConfig['Sigma'][iPt], 1)
             if fixSigma[iPt]:
                 vnFitter[iPt].FixSigmaFromMassFit()
+            # nSigma4SB
+            if 'NSigma4SB' in fitConfig:
+                vnFitter[iPt].SetNSigmaForVnSB(fitConfig['NSigma4SB'][iPt])
+                print(f'NSigma4SB = {fitConfig["NSigma4SB"][iPt]}')
```
Comment (stefanopolitano) [inline run3/flow/get_vn_vs_mass.py:365]: Hi @Libeiyu ! Thanks a lot for the work. I would remove the print here though

**Comment by Libeiyu** [comment]
Hi @stefanopolitano! Apologies for the multiple commits. They include unnecessary debugging code. I'm unsure how to merge them into one. The reflections for D0 have been added and tested with run2 MC. There is a `segmentation` violation`, but it's ok for the results.

File: run3/flow/get_vn_vs_mass.py (hunk 1 above)
Comment (Libeiyu) [reply run3/flow/get_vn_vs_mass.py:375]: Hi @stefanopolitano! I am very sorry for the late reply. Is there any update on this matter? If there are no other questions, I'd like to ask for approval of this PR, please.

**Pull request merged**